docs/source/fitting/fitfunctions/ChudleyElliotSQE.rst

```diff
@@ -0,0 +1,48 @@
+.. _func-Chudley-ElliotSQE:
+
+================
+ChudleyElliotSQE
+================
+
+.. index:: ChudleyElliotSQE
+
+Description
+-----------
+
+This fitting function models the dynamic structure factor
+for a particle undergoing jump diffusion using the Chudley-Elliot Jump diffusion model [1]_.
+
+.. math::
+
+   S(Q,E) = Height \cdot \frac{1}{\pi} \frac{\Gamma}{\Gamma^2+(E-Centre)^2}
+
+   \Gamma = \frac{\hbar}{\tau} \cdot \left(1 - \frac{sin(Ql)}{Ql}\right)
+
+where:
+
+-  :math:`Height` - Intensity scaling, a fit parameter
+-  :math:`Centre` - Centre of peak, a fit parameter
+-  :math:`Tau` - Residence time, a fit parameter
+-  :math:`l` - Jump lengths, a fit parameter
+-  :math:`Q` - Momentum transfer, an attribute (non-fitting)
+
+Units of :math:`l` are inverse units of :math:`Q`.
+
+Units of :math:`HWHM` are :math:`meV` if units of :math:`\tau` are *ps*.
+Alternatively, units of :math:`HWHM` are :math:`\mu eV` if units of
+:math:`\tau` are *ns*.
+
+.. attributes::
```

Comment:
When I build the html-documentation it creates a table with "Attributes  (non-fitting parameters)". It contains two rows: one with "Q" and another one with "WorkspaceIndex". Is that how it is supposed to look like and what is the meaning of "WorkspaceIndex"? If "Q" is in this table, wouldn't it make sense to have "E" in it , as well?

Reply:
I am not sure why `WorkspaceIndex is being added to that list. As for `E`, `E` is the x in this case, I don;t think we list the x axes in other fit functions.